datamodel.itop-standard-email-synchro.xml

```diff
@@ -833,7 +838,7 @@ EOF
 		}
 					
 		// Determine which field to update
-		$sAttCode = 'public_log';
+		$sAttCode = $this->Get('ticket_updates_log_attcode');
 		$aAttCodes = MetaModel::GetModuleSetting('itop-standard-email-synchro', 'ticket_log', array('UserRequest' => 'public_log', 'Incident' => 'public_log'));
 		if (array_key_exists(get_class($oTicket), $aAttCodes))
 		{
```

Comment:
`$sAttCode` will get overwritten by this default module setting..

Should this setting be obsoleted and ignored maybe?

Reply:
You are right, I'll fix it!

Reply:
I just changed the conf. param. default value to an empty array; but actually the whole conf. param. should be obsoleted you are right.

Reply:
That's why obsoleting the param. and removing the line from the code is the best way IMHO.

For existing users, we should check the existing conf. param. and apply the corresponding value to existing mail inboxes. I missed that!

Reply:
That could be an action during setup (after database creation) for example..